.github/workflows/inspektor-gadget.yml

```diff
@@ -991,6 +991,38 @@ jobs:
         name: Release ${{ github.ref }}
     - name: Get all artifacts.
       uses: actions/download-artifact@v3
+    - name: Setup go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ env.GO_VERSION }}
+        cache: true
+      id: go
+    - name: Build ig distributions packages
+      run: |
+        go install github.com/goreleaser/nfpm/v2/cmd/nfpm@latest
+
+        mkdir ig_packages
+        for ig_archive in ig-*-*-tar-gz/ig-*-*.tar.gz; do
+          cp .nfpm_template.yaml nfpm.yaml
+
+          mkdir archive_output
+          tar zxvf $ig_archive -C archive_output
+          export path=$(find archive_output -name 'ig')
+
+          # Fill the template file with corresponding information.
+          export arch=$(echo $ig_archive | cut -d'-' -f3)
+          perl -pi -e 's/ig_arch/$ENV{arch}/; s/ig_version/v0.14.0/; s/ig_path/$ENV{path}/' nfpm.yaml
```

Comment:
Can we use `./.github/actions/set-container-repo-and-determine-image-tag` to determine the version instead of hardcoding it?